types/cesium/index.d.ts

```diff
@@ -2134,8 +2134,9 @@ declare namespace Cesium {
         height: Property;
         scaleByDistance: Property;
         translucencyByDistance: Property;
-        disableDepthTestDistance: Property;
+        disableDepthTestDistance: Property | number;
```

Comment:
Constructor options for Graphics objects do accept raw types, though the documentation does not mention it. Additionally, you can set these values with the raw types. However, upon setting the value, the properties are always wrapped in a `Property` of some kind and cannot be accessed directly as the raw value. See https://cesium.com/docs/cesiumjs-ref-doc/BillboardGraphics.html?classFilter=billboardGraphics for the documentation and https://github.com/CesiumGS/cesium/blob/master/Source/DataSources/createPropertyDescriptor.js for the source code that wraps every graphics property into a `Property`.

Because writes to these properties can be either type, but reads are only one type, we should restrict these types to match whatever satisfies both, otherwise we lose typesafety on reads.

Reply:
It looks like having different `set` vs `get` types is a longstanding issue on Typescript, and until it is resolved, we have to support only the narrower type: https://github.com/microsoft/TypeScript/issues/2521